tokens/endpoint_test.go

```diff
@@ -18,27 +17,36 @@ import (
 
 	"storj.io/common/testcontext"
 	"storj.io/common/uuid"
+	"storj.io/private/dbutil/pgtest"
 	"storj.io/storjscan/api"
+	"storj.io/storjscan/blockchain"
 	"storj.io/storjscan/private/testeth"
 	"storj.io/storjscan/private/testeth/testtoken"
+	"storj.io/storjscan/storjscandb/storjscandbtest"
 	"storj.io/storjscan/tokens"
 )
 
-func TestEndpointWithAuth(t *testing.T) {
+func TestEndpoint(t *testing.T) {
```

Comment:
Test fails, the API key is missing from the header.
Error Trace:	endpoint_test.go:80
        	            				testeth.go:37
        	Error:      	Not equal: 
        	            	expected: 200
        	            	actual  : 401

Reply:
Might be my mistake during rebase